### PR TITLE
Update the CHANGELOG in preparation for next release

### PR DESCRIPTION
**Pull request by tsandall**
Signed-off-by: Torin Sandall <torinsandall@gmail.com>

### CODE REVIEW

**Review by anderseknert — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,62 @@
...
 project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+
+This release contains a number of enhancements and bug fixes.
+
+### Tooling
+
+- The `eval` subcommand now supports a `-s`/`--schema` flag that accepts a JSON schema for the `input` document. The schema is used when type checking the policy so that invalid references to (or operations on) `input` data are caught at compile time. In the future, the schema support will be expanded to accept multiple schemas and rule-level annotations. See the new [Schemas](https://www.openpolicyagent.org/docs/edge/schemas/) documentation for details. Authored by @[aavarghese](https://github.com/aavarghese) and @[vazirim](https://github.com/vazirim).
+- The `eval`, `test`, `bench` and REPL subcommands now supports a `-t`/`--target` flag to set the evaluation engine to use. The default engine is `rego` referring to the standard Rego interpreter in OPA. Users can now select `wasm` to enable Wasm compilation and execution of policies ([#2878](https://github.com/open-policy-agent/opa/issues/2878)).
```
Comment (srenatus) [inline CHANGELOG.md:13]: IIRC we've remove the `-t` shortcut for `--timeout` in `opa test`. Worth mentioning, WDYT?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,62 @@
...
+- The `eval` subcommand now supports a `raw` option for `-f`/`--format` that is useful in bash scripts. Authored by @[jaspervdj-luminal](https://github.com/jaspervdj-luminal).
```
Comment (srenatus) [inline CHANGELOG.md:14]: 👏 @jaspervdj-luminal

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,62 @@
...
+### WebAssembly
+
+- The `data` document no longer needs to be initialized to an empty object ([#3130](https://github.com/open-policy-agent/opa/issues/3130)).
+- The mpd library is now initalized by the module's `Start` function ([#3110](https://github.com/open-policy-agent/opa/issues/3110)).
+- The planner now longer re-plans rules blindly when `with` statements are encountered ([#3150](https://github.com/open-policy-agent/opa/issues/3150)).
+- The planner and compiler now support dynamic dispatch. Previously the planner would enumerate all functions and invocation was controlled at runtime ([#2936](https://github.com/open-policy-agent/opa/issues/2936)).
+- The compiler now inserts memoization instructions into function bodies instead of at callsites. This reduces the number of wasm instructions in the resulting binary ([#3169](https://github.com/open-policy-agent/opa/pull/3169)).
+- The wasmtime runtime is now the default runtime used by OPA to execute compiled policies. The new runtime no longer leaks memory when policies are reloaded.
+- The planner and compiler now intern strings and booleans and implement a few imcro-optimizations to reduce the size of the resulting binary.
```
Comment (srenatus) [inline CHANGELOG.md:54]:
```suggestion
- The planner and compiler now intern strings and booleans and implement a few micro-optimizations to reduce the size of the resulting binary.
```

File: CHANGELOG.md (hunk 1 above)
Comment (srenatus) [reply CHANGELOG.md:13]: Ignore me, it's mentioned below.

**Review by srenatus — APPROVED** [review]
LGTM, found one typo.

**Review by srenatus — APPROVED** [review]

**Pull request merged**